Documentation~/handles.md

```diff
@@ -0,0 +1,56 @@
+# Cinemachine Handle toolbar
+
+The Cinemachine Handle toolbar is a group of 3D controls that allow you to manipulate virtual camera parameters visually in the Scene view. You can use the handle tools to interactively adjust the selected object's parameters quickly and efficiently rather than typing numbers into the inspector.
+
+## Activating and deactivating the toolbar
+
+To activate the Handle toolbar:
+
+* Right-click on the **Scene** tab in the Scene view.
+* Select **Overlays** and then **Cinemachine** from the pop-up menu.
+
+![overlays-menu](images/overlays-menu.png)
+
+The Cinemachine Handle toolbar appears as shown above.
+* Select any Handle tool from the toolbar.
+* The corresponding Handle then appears in the Scene view.
+* You can then select the Handle again to deactivate it from the Scene view.
+
+## Handle tools
+
+The following four Handle tools are available in the toolbar:
+
+![handle-toolbar](images/handle-toolbar.png)
+
+1. **Field of View (FOV)**
+
+The camera view in vertical degrees. You can drag the points to increase or decrease the horizontal or vertical FOV.
```

Comment:
FOV tool can control Vertical FOV, Horizontal FOV, Orthographic Size, or Focal Length depending on what is selected by the user.

If the camera is in Perspective mode -> Vertical FOV or Horizontal FOV (depending on their selection in the Main Camera)
If the camera is in Orthographic mode -> Orthographic Size
If the camers is in Physical mode -> Focal Length